behandlingslager/domene/src/main/java/no/nav/ung/sak/behandlingslager/behandling/startdato/UngdomsytelseBekreftetPeriodeEndring.java

```diff
@@ -0,0 +1,93 @@
+package no.nav.ung.sak.behandlingslager.behandling.startdato;
+
+import jakarta.persistence.*;
+import no.nav.ung.kodeverk.ungdomsytelse.UngdomsytelseSatsTypeKodeverdiConverter;
+import no.nav.ung.kodeverk.ungdomsytelse.periodeendring.UngdomsytelsePeirodeEndringTypeKodeverdiConverter;
+import no.nav.ung.kodeverk.ungdomsytelse.periodeendring.UngdomsytelsePeriodeEndringType;
+import no.nav.ung.sak.behandlingslager.BaseEntitet;
+import no.nav.ung.sak.typer.JournalpostId;
+import org.hibernate.annotations.Immutable;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import static no.nav.ung.sak.behandlingslager.behandling.startdato.VurdertSøktPeriode.SøktPeriodeData;
+
+@Entity(name = "UngdomsytelseBekreftetPeriodeEndring")
+@Table(name = "UNG_BEKREFTET_PERIODE_ENDRING")
+@Immutable
+public class UngdomsytelseBekreftetPeriodeEndring extends BaseEntitet {
```

Comment:
Er det riktigere å kalle denne for Ungdomsprogram* da det er vel egentlig periode for ungdomsprogrammet veileder legger inn?

Reply:
Vi har brukt Ungdomsytelse som prefiks for alt som er spesifikt for ytelsen, men eg trur ikkje nokon blir forvirret om eg prefikser med Ungdomsprogram heller.